.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 
 rvm:
-  - 2.2.3
+  - 2.2.5
```

Comment:
Thanks. How did you determine that `ruby_dep` is a dependency of `guard`? Also, according to https://rubygems.org/gems/ruby_dep/versions/1.4.0, it seems to support anything `>= 2.2` (because of `~> 2.2`)? Or `>= 2.2.5` has higher priority?